backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -30,7 +30,7 @@ object BackendWorkflowInitializationActor {
   */
 trait BackendWorkflowInitializationActor extends BackendLifecycleActor with ActorLogging {
```

Comment:
Is this name okay? Since it does more than just initialization (like abort et. al) ?